run-tests.py

```diff
@@ -100,7 +100,7 @@ def run_pytest(display):
 
 def main():
     check_dependencies()
-    display = get_open_display()
+    display = get_open_display() + 1
```

Comment:
This seems like a bug in the function that should be fixed there.

I assume that whenever an X display is active, it puts a `.X{number}-lock` in `/tmp`. It does this on Fedora 27 workstation, but I'm not sure about the container environment.